api/spec/src/billing/invoices.tsp

```diff
@@ -263,6 +271,23 @@ interface CustomerInvoiceEndpoints {
 
   // Line management
 
+  /**
+   * Get an invoice line
+   */
+  @get
+  @route("/lines/{lineId}")
+  @summary("Get an invoice line")
+  @operationId("billingGetInvoiceLine")
+  @extension("x-internal", true)
+  getInvoiceLine(
+    @path customerId: ULID,
+    @path invoiceId: ULID,
+    @path lineId: ULID,
+  ): {
+    @statusCode _: 204;
```

Comment:
No need for statusCode, as it should be 200? Like: Invoices.InvoiceLine | OpenMeter.NotFoundError